### ISSUE TITLE
complete update of gold generation scripts

### ISSUE THREAD

**Issue by bmcdanie** [project-asgard/DG-SparseGrid#53]
I've fixed/updated most of the gold generation scripts over time, but `gold_pde` is still broken. I need to fix it once I get done with my current work on non-uniform level.

**Comment by hbrunie** [comment]
@bmcdanie is this still open? As I need to update `gold_pde.m` for the ode renaming in Asgard (cf issue #63 ), maybe I can work on this issue too?

**Comment by bmcdanie** [comment]
yes, I believe so. much of the MATLAB was updated since the gold scripts were written, and I had to update them to still work/produce the correct data. I think `gold_pde` still needs update, yes.
